src/Form/Extension/ApiProductTypeExtension.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Loevgaard\SyliusBrandPlugin\Form\Extension;
```

Comment:
Should this be another namespace. Does Sylius put API type extension here or in `...\Extension\Api` or something?

Reply:
`Sylius` just have two different types in two different Bundles (`AdminBundle` and `AdminApiBundle`). So I've used name like that. But yeah, moving it to `Extension\Api` will be better solution.

Reply:
Okay, let's do that then